index.html

```diff
@@ -1186,7 +1186,7 @@ <h1>Data Model</h1>
           </td>
           <td>
 A finite ordered sequence of key/value pairs, with no key appearing twice as
-specified in [[INFRA]].
+specified in [[INFRA]]. Sometimes referred to as just "map".
```

Comment:
We really should be using 'map' throughout the spec instead. I'll make that change later.